EVASEAC/app/screens/protocol3/Pt3FormScreen.tsx

```diff
@@ -0,0 +1,433 @@
+import { useState } from "react";
+import { FlatList, StyleSheet, Text, View } from 'react-native';
+import DropDownPicker from "react-native-dropdown-picker";
+
+interface Pt3FormProps {
+    navigation : NavigationProp<any, any>;
+}
+
+// General TODOs:
+// 2. See more about FlatList. Maybe this can give us a solution for rendering each row
+// 3. Fix items error (Dropdown picker)
+// 4. Fix two dropdowns open at the same time. Better UX behaviour should close other open dropdown when one new it's open
+// 5. Fix overlay open dropdown
+// 5.1. Fix scroll down inside dropdown. See: https://github.com/hossein-zare/react-native-dropdown-picker/issues/57
+
+const Pt3FormScreen = (props:Pt3FormProps) => {
+    // TODO: Refactor this, I hardcoded it, it should be a better way to implement this
+    // Opens
+    const [openDrop1, setOpenDrop1] = useState(false);
+    const [openDrop2, setOpenDrop2] = useState(false);
+    const [openDrop3, setOpenDrop3] = useState(false);
+    const [openDrop4, setOpenDrop4] = useState(false);
+    const [openDrop5, setOpenDrop5] = useState(false);
+    const [openDrop6, setOpenDrop6] = useState(false);
+    const [openDrop7, setOpenDrop7] = useState(false);
+    const [openDrop8, setOpenDrop8] = useState(false);
+    const [openDrop9, setOpenDrop9] = useState(false);
+    const [openDrop10, setOpenDrop10] = useState(false);
+    const [openDrop11, setOpenDrop11] = useState(false);
+    const [openDrop12, setOpenDrop12] = useState(false);
+    const [openDrop13, setOpenDrop13] = useState(false);
+    const [openDrop14, setOpenDrop14] = useState(false);
+    const [openDrop15, setOpenDrop15] = useState(false);
+    const [openDrop16, setOpenDrop16] = useState(false);
+    const [openDrop17, setOpenDrop17] = useState(false);
+    const [openDrop18, setOpenDrop18] = useState(false);
+    const [openDrop19, setOpenDrop19] = useState(false);
+    const [openDrop20, setOpenDrop20] = useState(false);
+
+    // Values
+    const [value1, setValue1] = useState(false);
+    const [value2, setValue2] = useState(false);
+    const [value3, setValue3] = useState(false);
+    const [value4, setValue4] = useState(false);
+    const [value5, setValue5] = useState(false);
+    const [value6, setValue6] = useState(false);
+    const [value7, setValue7] = useState(false);
+    const [value8, setValue8] = useState(false);
+    const [value9, setValue9] = useState(false);
+    const [value10, setValue10] = useState(false);
+    const [value11, setValue11] = useState(false);
+    const [value12, setValue12] = useState(false);
+    const [value13, setValue13] = useState(false);
+    const [value14, setValue14] = useState(false);
+    const [value15, setValue15] = useState(false);
+    const [value16, setValue16] = useState(false);
+    const [value17, setValue17] = useState(false);
+    const [value18, setValue18] = useState(false);
+    const [value19, setValue19] = useState(false);
+    const [value20, setValue20] = useState(false);
```

Comment:
I don't know if leave this as a TODO or try to find a better way to represent this. Maybe as a data structure or an array.